### PR TITLE
Release note about targets built in the Windows packages

### PR DESCRIPTION
**Pull request by zmodem**
LLVM_TARGETS_TO_BUILD was set in #106059

### CODE REVIEW

**Review by tru — APPROVED** [review]

**Comment by zmodem** [comment]
I think you need to push the merge button, I don't have permission.

**Comment by tru** [comment]
Yeah I'll merge it - I just do it in batches when I have time to review and integrate them all (I can't use the merge button either - you have to do it with a script ... due to reasons ...)

**Pull request merged**
